app/boards/shields/splitkb_aurora_lily58/splitkb_aurora_lily58.conf

```diff
@@ -0,0 +1,9 @@
+# Uncomment these two line to add support for encoders to your firmware
+# CONFIG_EC11=y
+# CONFIG_EC11_TRIGGER_GLOBAL_THREAD=y
+
+# Uncomment the following line to enable the Kyria OLED Display
```

Comment:
```suggestion
# Uncomment the following line to enable the OLED Display
```

Reply:
Can replace with `Lily58`, also just noticed the same with the Aurora Corne shield.

Reply:
Good catch. Fixed.